config/config.yaml

```diff
@@ -124,3 +124,15 @@ voc:
     - E484K
     - K417N
     - H69_V70del
+
+# assemblers used for assembly comparison
+assemblers_for_comparison:
+  - megahit-std
```

Comment:
Maybe we should include some of the assemblers in the .tests config and include the assembler comparison in GitHub Actions.